pkg/time/time.go

```diff
@@ -57,7 +74,7 @@ func add(l *lua.State) int {
 	hour := lua.OptInteger(l, -3, 0)
 	l.Pop(3)
 
-	inc := startUnix.Add(time.Hour * time.Duration(hour) + time.Minute * time.Duration(minute) + time.Second * time.Duration(second))
```

Comment:
Can you revert this back to having spaces, it makes the code a lot easier to read.